micro-rdk-server/.cargo/config.toml

```diff
@@ -18,9 +18,3 @@ rustflags = ["-C", "default-linker-libraries"]
 linker = "ldproxy"
 runner = "espflash --monitor"
 rustflags = ["-C", "default-linker-libraries"]
-
-
-[env]
-ESP_IDF_TOOLS_INSTALL_DIR = {value = "fromenv", force = true}
-ESP_IDF_VERSION= { value = "v4.4.1", force = true}
-MCU = { value= "esp32", force = true}
```

Comment:
I've stopped setting `MCU` explicitly because the docs say it is derived from the target triple.